README.md

```diff
@@ -0,0 +1,62 @@
+# Highlighter
+Functionality for OpenStax note taking features including:
+- serializing highlights
+- deserializing highlights
+- versioned serialization strategies
+- extracting highlighted content
+- loading wrappers for styling highligted content
+- managing focus/scrolling to highlights
+
+## Setup
+
+install [nvm](https://github.com/creationix/nvm#installation)
+
+```bash
+# use the right version of node
+nvm install
+
+# install yarn, skip if you have it already
+npm install -g yarn
+
+# install dependencies
+yarn
+```
```

Comment:
I think `yarn` recommends running `yarn install` just in case the default action for bare `yarn` changes in the future